Automattic/pocket-casts-ios
# Ratings: adjustments

* Change minimum number of stars to 1 (instead of half)
* Update the stars design under the podcast screen
* Fixes the warning of showing the ratings screen multiple times

## To test

1. Go to Profile > Settings > Beta features > enable `giveRatings`
2. ✅ Go to any podcast and check that the stars appears correctly
3. Tap "Rate"
4. ✅ The Rating view should appear
5. Go back
6. Tap the "..." button (on the right of the search bar)
7. Change Group Episodes to something else
8. Tap "Rate"
9. Check the console log, it shouldn't have any warning
10. Disable the flag
11. ✅ Everything should work as it was before

## Checklist
